test/integration/restart_task_test.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+require 'test_helper'
+require 'kubernetes-deploy/restart_task'
+
+class RestartTaskTest < KubernetesDeploy::IntegrationTest
+  def test_restart
+    deploy_fixtures("basic", subset: ["configmap-data.yml", "web.yml.erb"])
+
+    deployment = fetch_deployment("web")
+    refute fetch_restarted_at(deployment), "no RESTARTED_AT env on fresh deployment"
+
+    restart = KubernetesDeploy::RestartTask.new(
+      context: KubeclientHelper::MINIKUBE_CONTEXT,
+      namespace: @namespace,
+    )
```

Comment:
I didn't want to put this to `setup` because the arguments to `RestartTask.new` are different across test cases.